addOns/reports/src/main/zapHomeFiles/reports/traditional-xml-plus/report.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<OWASPZAPReport
+	th:attr="version=${zapVersion}, generated=${generatedString}">
+	<th:block th:each="site: ${reportData.sites}">
+		<site th:name="${site}"
+			th:attr="host=${helper.getHostForSite(site)}, port=${helper.getPortForSite(site)}, ssl=${helper.isSslSite(site)}">
+			<alerts>
+				<th:block
+					th:each="alert: ${helper.getAlertsForSite(alertTree, site)}"
+					th:with="instances=${helper.getAlertInstancesForSite(alertTree, site, alert.name, alert.risk)}">
+					<alertitem>
+						<pluginid th:text="${alert.pluginId}"></pluginid>
+						<alertRef th:text="${alert.alertRef}"></alertRef>
+						<alert th:text="${alert.name}"></alert>
+						<name th:text="${alert.name}"></name>
+						<riskcode th:text="${alert.risk}"></riskcode>
+						<confidence th:text="${alert.confidence}"></confidence>
+						<riskdesc
+							th:text="${helper.getRiskString(alert.risk) + ' (' + helper.getConfidenceString(alert.confidence) + ')'}"></riskdesc>
+						<confidencedesc
+							th:text="${helper.getConfidenceString(alert.confidence)}"></confidencedesc>
+						<desc
+							th:text="${helper.legacyEscapeParagraph(alert.description)}"></desc>
```

Comment:
We should stop adding templates using the legacy escaping.

Reply:
I copied the original XML report, as I want the new report to be compatible with the basic XML report. Does the new escaping function generate equivalent results? And what is the function that I should use? In org.zaproxy.addon.reports.ReportHelper, I could only find the legacy escaping functions.

Thanks!

Reply:
Don't use any function, let Thymleaf do the pertinent escaping.

Reply:
+1 to remove `legacyEscapeParagraph` - that is just so that the 'legacy' reports generated by thymleaf matched the previous ones, new reports should not use them

Reply:
I removed all `legacyEscape*` from the report and updated the example report to match the new behavior.